www-servers/nginx/nginx-1.29.0-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
Just git mv nginx for this change, especially as you're changing the init script in-place (which is fine here).